src/components/tabs/_tabs.scss

```
@@ -43,11 +43,10 @@
     border-radius: 3px 3px 0 0;
     color: var(--ons-color-text);
     display: inline-block;
-    height: 2.55rem;
-    line-height: 2.25rem;
+    height: 2.5rem;
     margin: 0 0.5rem 0 0;
     overflow: visible;
-    padding: 0 1rem;
+    padding: 0.3125rem 1rem 0.375rem;
```

I think this can be set to this and this causes less changes to the tabs component. It also means that we can stay away from specifically sized padding like this and can align the padding here with our padding size scale
```suggestion
    padding: 0.25rem 1rem 0.25rem;
```